notebooks/Part 2 - Model Development.ipynb

```diff
@@ -1 +1 @@
rible, horrible customer service, I have never met such rude people. Why is it so bad? Would not recommend at all.',\n    'Everything I ordered arrived on time and looked exactly like in the pictures! This company has high quality products.',\n    'I bought somethings on sale, they were a great deal. Will be buying more next time.'\n]\n\noverall_sentiment = group_comments_by_sentiment(comments)\nsign_word_to_symbol = {Sign.NEGATIVE: '-', Sign.POSITIVE: '+', Sign.NEUTRAL: '='}\n\n# print out results in readible way\nfor sign in overall_sentiment:\n    print('{}:'.format(sign))\n    for comment in overall_sentiment[sign]:\n        sentence_printout_text = ''\n        for sentence_sign, sentence in comment:\n            sentence_printout_text += '({}) {}\\n'.format(sign_word_to_symbol[sentence_sign], sentence)\n        print(sentence_printout_text)", "execution_count": 22, "outputs": [{"output_type": "stream", "text": "Negative:\n(-) I bought several items: socks, shirt, sweater.\n(-) By far my most favorite was the shirt because it is so soft.\n(-) However, the sweater and socks missed the mark.\n\n(-) Horrible, horrible customer service, I have never met such rude people.\n(-) Why is it so bad?\n(-) Would not recommend at all.\n\nPositive:\n(-) My order arrived several days late.\n(+) But when I contacted customer serivce they were very helpful and refunded me.\n\n(+) I bought somethings on sale, they were a great deal.\n(=) Will be buying more next time.\n\nNeutral:\n(-) Everything I ordered arrived on time and looked exactly like in the pictures!\n(+) This company has high quality products.\n\n", "name": "stdout"}]}, {"metadata": {}, "cell_type": "markdown", "source": "<a id=\"authors\"></a> \n### Authors\nThis notebook was created by the [Center for Open-Source Data & AI Technologies](http://codait.org).\n\nCopyright \u00a9 2020 IBM. This notebook and its source code are released under the terms of the MIT License."}, {"metadata": {}, "cell_type": "markdown", "source": "<div style=\"background:#F5F7FA; height:110px; padding: 2em; font-size:14px;\">\n<span style=\"font-size:18px;color:#152935;\">Love this notebook? </span>\n<span style=\"font-size:15px;color:#152935;float:right;margin-right:40px;\">Don't have an account yet?</span><br>\n<span style=\"color:#5A6872;\">Share it with your colleagues and help them discover the power of Watson Studio!</span>\n<span style=\"border: 1px solid #3d70b2;padding:8px;float:right;margin-right:40px; color:#3d70b2;\"><a href=\"https://ibm.co/wsnotebooks\" target=\"_blank\" style=\"color: #3d70b2;text-decoration: none;\">Sign Up</a></span><br>\n</div"}], "metadata": {"kernelspec": {"name": "python3", "display_name": "Python 3.7", "language": "python"}, "language_info": {"name": "python", "version": "3.7.9", "mimetype": "text/x-python", "codemirror_mode": {"name": "ipython", "version": 3}, "pygments_lexer": "ipython3", "nbconvert_exporter": "python", "file_extension": ".py"}}, "nbformat": 4, "nbformat_minor": 4}
```

Comment:
<p>Add explanation:</p><p> </p><p>In this first implementation (calculate_unigram_sentiment()), we use the unigrams dataset, which contains the sentiment of various unigrams. Given an input sentence, we tokenize the sentence (break it up as a list of words) then match each word in the sentence to the words in the unigram dataset and find its sentiment value. If the word is not found in the dataset, it is skipped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>done</p>